runners/behat/before_script.sh

```diff
@@ -4,7 +4,7 @@
 set -e $DRUPAL_TI_DEBUG
 
 # Ensure the right Drupal version is installed.
-drupal_ti_ensure_drupal
+drupal_ti_ensure_drupal_installed
```

Comment:
However I think I would rather do it the other way round for BC reasons,

e.g. drupal_ti_ensure_drupal_downloaded + drupal_ti_ensure_drupal